### PR TITLE
fix: various UI issues on project page

### PR DESCRIPTION
**Pull request by blushi**
## Description

Closes: regen-network/regen-registry#1584

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

- [x] provided a link to the relevant issue or specification
- [x] provided instructions on how to test
- [x] reviewed "Files changed" and left comments if necessary
- [x] confirmed all CI checks have passed

### How to test

https://deploy-preview-1825--regen-registry.netlify.app/project/C52-001

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...

- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed code correctness and readability
- [ ] verified React components follow DRY principles
- [ ] reviewed documentation is accurate
- [ ] reviewed tests
- [ ] manually tested (if applicable)

### CODE REVIEW

**Comment by blushi** [comment]
@erikalogie you can test this on https://deploy-preview-1825--regen-registry.netlify.app/project/C52-001

File: web-registry/src/components/organisms/ProjectTopSection/ProjectTopSection.tsx [hunk 1]
```diff
@@ -185,18 +185,20 @@ function ProjectTopSection({
...
-              description={
-                <BlockContent
-                  content={creditClassSanity?.shortDescriptionRaw}
-                />
-              }
-              imgSrc={getSanityImgSrc(creditClassSanity?.image)}
-              sx={{ mt: [2, 4], py: [2, 6] }}
-            />
-          </Link>
+          {creditClassSanity && (
```
Comment (flagrede) [inline web-registry/src/components/organisms/ProjectTopSection/ProjectTopSection.tsx:188]: Maybe we could `WithLoader` with the `skeleton variant to reduce layout shift a bit?

**Review by flagrede — APPROVED** [review]

File: web-registry/src/components/organisms/ProjectTopSection/ProjectTopSection.tsx (hunk 1 above)
Comment (blushi) [reply web-registry/src/components/organisms/ProjectTopSection/ProjectTopSection.tsx:188]: if there's no sanity credit class, we don't want to display it the card all, so we wouldn't want to display the skeleton at all in this case

Comment (flagrede) [reply web-registry/src/components/organisms/ProjectTopSection/ProjectTopSection.tsx:188]: Make sense, we could plug it to`isFetching` flag from `react-query` instead then but then we would end up showing a skeleton without data after in some cases which could be weird.
Let's leave it like that hen.

**Pull request merged**